shared/actions/team-building.tsx

```diff
@@ -142,7 +142,7 @@ const search = (state: TypedState, {payload: {namespace}}: NSAction) => {
 
   const query = teamBuildingSearchQuery
   let impTofuQuery: RPCTypes.ImpTofuQuery | null = null
-  if (flags.sbsContacts && teamBuildingSelectedService === 'keybase') {
+  if (flags.sbsContacts && teamBuildingSelectedService === 'keybase' && namespace == 'chat2') {
```

Comment:
this file should be namespace agnostic, no chat2 specific stuff should be in here

Reply:
What do you think the best way to only get contacts on New Chat searches is? It seems like it would be bad to split this into two files, but I need to have some logic somewhere in here that acts different in one case vs the other.